spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -0,0 +1,8 @@
+module BlockScore
+  class Collection
+    RSpec.describe Member do
+      describe '.save' do
+      end
+    end
```

Comment:
Any plans for this file?  Also, to save yourself from nesting hell, you can always write this like so:

```ruby
RSpec.describe Member, '.save' do
end
```